src/index.js

```diff
@@ -1,14 +1,33 @@
 const React = require('react');
-const codemirror = require('./codemirror.jsx');
+const codemirror = require('./codeMirror/index.jsx');
+const codeEditor = require('./codeEditor/index.jsx');
 
 // eslint-disable-next-line react/display-name
-module.exports = (code, lang, opts = { dark: false, tokenizeVariables: false }) =>
-  React.createElement(
-    'span',
-    {
-      className: opts.dark ? 'cm-s-tomorrow-night' : 'cm-s-neo',
-    },
+module.exports = (code, lang, opts = { dark: false, tokenizeVariables: false, editable: false }, editorProps = {}) => {
+  if (opts.editable) {
+    return React.createElement(
+      codeEditor,
+      {
+        ...editorProps,
+        code,
+        lang,
+      },
+      null
+    );
+  }
+
+  let classes = '';
+  if (opts.highlightMode) {
+    classes = 'CodeEditor cm-s-material-palenight';
```

Comment:
If `editable` is enabled, should these classes be applied to that element?

Reply:
The `editable` flag transitions the output to use the full `react-codemirror2` export, which has a bunch of styling baked in. We don't need to apply the same classes to both.